include/swift/Runtime/Debug.h

```diff
@@ -102,7 +102,7 @@ swift_dynamicCastFailure(const void *sourceType, const char *sourceName,
                          const char *message = nullptr);
 
 extern "C"
-void swift_reportError(const char *message);
+void swift_reportError(int isDebugAssertConfiguration, const char *message);
```

Comment:
`swift_reportError` might be ABI. It would be a bit more future-proof to introduce this as a general `flags` attribute, with a `showBacktrace` bit (and the rest of the bits reserved for future expansion).